### PR TITLE
fix: test curriculum challenges

### PR DESCRIPTION
**Pull request by ValeraS**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

tests only challenges for the English language

### CODE REVIEW

File: curriculum/test-challenges.js [hunk 1]
```diff
@@ -189,18 +196,24 @@ function createTest({
...
-Observable.from(getChallenges())
-  .do(({ challenges }) => {
-    challenges.forEach(challenge => {
-      const result = validateChallenge(challenge);
-      if (result.error) {
-        console.log(result.value);
-        throw new Error(result.error);
-      }
-    });
+Observable.fromPromise(getChallengesForLang('english'))
```
Comment (raisedadead) [inline curriculum/test-challenges.js]:
```suggestion
Observable.fromPromise(getChallengesForLang(LOCALE || 'english'))
```

We can derive the locale variable from the `.env`, let's just make it default.

**Review by raisedadead — CHANGES_REQUESTED** [review]

File: curriculum/test-challenges.js (hunk 1 above)
Comment (ValeraS) [reply curriculum/test-challenges.js]: Do we need to test all languages on CI/CD?

Comment (raisedadead) [reply curriculum/test-challenges.js]: Yes, the languages are essentially forks of the english version, and as a matter of fact will always be behind the english versions. Meaning broken tests will have to be fixed in every language version again.

Also, the CD will be responsible for deploying the language versions of the applications to their own "hosting" locations. 

We would build these in parallel as per the language (actually its more complicated, I am over simplifying the workflow a bit).

Comment (ValeraS) [reply curriculum/test-challenges.js]: fixed

**Review by raisedadead — APPROVED** [review]
Nice work as usual @ValeraS !

**Comment by raisedadead** [comment]
I am going to pull this down to test on my fork for the CI. Other than that we are good to go.

**Comment by raisedadead** [comment]
@ValeraS looks like this is great.

https://github.com/raisedadead/freeCodeCamp/pull/19

**Comment by raisedadead** [comment]
The build seems to have caught all that is required @Bouncey 	

@ValeraS 

IDK, and this not a concern of your PR per say, but can we get rid of the stack somehow:

https://travis-ci.org/raisedadead/freeCodeCamp/builds/444145777#L5144-L5156

@RandellDawson @scissorsneedfoodtoo @beaucarnes 
We would need to go thru the build and fix the linting in challenge tests. 

/cc @tchaffee

**Comment by scissorsneedfoodtoo** [comment]
Everything LGTM! Awesome job, @ValeraS. Just ran the test locally and everything passes without a hitch.

**Pull request merged**